src/content/docs/security.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Security
+description: Security
+---
+
+Shorebird takes security very seriously. We take several steps to make sure that
+we protect the data you give us, that only you can publish changes to your app,
+and that the patches your users get are the same ones you created.
+
+## Release Artifact Privacy
+
+Your release artifacts (created by `shorebird release`) are hosted privately on
+GCP and are not publicly visible. When they need to be downloaded by you (to run
+`shorebird preview`, for example), we generate a signed URL that requires your
+auth credentials and expires after a short period of time.
+
+## Patch Integrity
+
+The most common security question we get is: How do you verify that the patches
+our users are downloading are the ones that I created and that they haven't been
+tampered with?
+
+We ensure the integrity of patches in several ways.
+
+Over the air:
+
+1. Patches are not full artifacts; rather, they are binary diffs. To create a
+   patch, you need access to the release artifact, which is private.
```

Comment:
Release's made to the stores are public, so it's easy to get that release artifact.